SITCOM-2111 Create a notebook to compare meteoblue data with real temperature data

b1quint opened the pull request.

b1quint committed:
Add new notebook for temperature forecast evaluation SITCOM-2111

ltoribiosc commented:
Delete `import pandas as pd` of the `get_forecast_closest_to_publication` function.  I don't know if you want to leave this function in the code but in the notebook it is not used.

ltoribiosc commented:
Remove 
```
    import pandas as pd
    from astropy.time import Time
```
of the `get_latest_forecast` function

ltoribiosc commented:
This is only a question: What does it mean that the forecast is published twice a day? Does it mean there are two forecasts for the same time period, or is new forecast data published twice a day but for different future times? If it's just updated twice daily with new information for future times, I understand how the code works. But if there are actually two forecasts for the same time (e.g., one in the morning and one in the evening), how does the code decide which one to use? Wouldn't it always pick the one published earlier in the day?

b1quint committed:
Update notebook associate with SITCOM-2111 based on comments

- Delete unused ``plot_hourly_weather_forecast``
- Delete unused ``query_daily_weather_forecast``
- Remove imports ``inside get_latest_forecast``
- Delete unused ``get_forecast_closest_to_publication``
